scoring/score.py

```diff
@@ -2,15 +2,66 @@ class InvalidScoresheetException(Exception):
     pass
 
 
+CAN_STATE_SCORES = {
+    "B": 3,  # 3 points for can with tape on the Bottom
+    "T": 1,  # 1 point for can with tape on the Top
+    "S": 0,  # 0 points for can on its Side
+}
+
+MAX_CANS = 28
+
+
 class Scorer(object):
     def __init__(self, teams_data, arena_data):
         self._teams_data = teams_data
+        self._arena_data = arena_data
 
     def calculate_scores(self):
-        raise NotImplementedError
+        scores = {}
+        for tla, info in self._teams_data.items():
+            zone = info['zone']
+            cans = self._arena_data[zone]['tokens']
```

Comment:
Could we use the term `cans` over `tokens`?

Reply:
In the raw data? We initially had this, however it involves much more involved changes to the scorer UI. How important do you feel this is?